nifi-nar-bundles/nifi-marklogic-bundle/NOTICE.txt

```diff
@@ -0,0 +1,2106 @@
+MarkLogic NiFi Processors
+
+
+Copyright � 2018 MarkLogic Corporation.
+
+This project is licensed under the Apache License, Version 2.0 (the "License"); you may not use this project except in compliance with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+
+Please direct questions, comments and requests to fossreview@marklogic.com. 
+
+Open source software required to be made available under license is included herein. In the event you are unable to obtain a copy of such open source software, please contact fossreview@marklogic.com and a copy will be made available to you.
+
+
+The following software may be included in this project (last updated May 1, 2018):

```

Comment:
This line can be removed.  The NOTICE needs to reflect what is actually bundled when this NAR is built and should account for what is in it that required NOTICE entries no more no less.